zuulcilint/__main__.py

```diff
@@ -101,39 +101,70 @@ def get_all_jobs(zuul_yaml_files: list[pathlib.Path]) -> list[list[str]]:
     return all_jobs
 
 
-def print_warnings(bad_yml_files: list[str], duplicated_jobs: set[str]) -> None:
+def print_warnings(bad_yml_files: list[str], duplicated_jobs: set[str], severity="warning") -> None:
     """Print warnings."""
-    nr_warnings = len(bad_yml_files) + len(duplicated_jobs)
-    print(f"Total warnings: {nr_warnings}")
+    n_bad = len(bad_yml_files)
+    n_dup = len(duplicated_jobs)
+
+    if(n_bad + n_dup == 0):
```

Comment:
Is this a micro optimisation? I think n_bad == 0 and n_dup == 0 might be easier to understand at a glance, despite being longer.